### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by julien-duponchelle** [julien-duponchelle/python-mysql-replication#492] [General]
I know the community is interested and I have started to setup the project. 

The label is OK and we have the contributing.md:
![image](https://github.com/julien-duponchelle/python-mysql-replication/assets/345437/fa2c27af-2dbb-44cc-8471-dd78c4e73a44)

**Comment by julien-duponchelle** [comment]
We could create some issues around improving the documentation and adding samples.

**Reply by dongwook-chan** [reply]
Sure. I will create issues labeled 'hacktoberfest'.

**Reply by dongwook-chan** [reply]
@julien-duponchelle 
I believe it would be beneficial for me and @sean-k1 to have maintainer rights on Read the Docs to ensure that our documentation stays updated and in sync with the changes we make to the codebase.

1. https://readthedocs.org/dashboard/
2. click 'python-mysql-replication'
3. click 'admin'
4. click 'maintainer'
5. add by email:
uhs20000@gmail.com
dongwook.chan@gmail.com

**Comment by julien-duponchelle** [comment]
@dongwook-chan can you confirm it's OK ?

**Reply by dongwook-chan** [reply]
@julien-duponchelle Perfect!
